zerver/webhooks/slack/doc.md

```diff
@@ -50,6 +50,12 @@ See also the [Zulip Slack incoming webhook integration][1].
    `BOT_OAUTH_TOKEN`, and add it to the end of the URL you generated
    above as: `&slack_app_token=BOT_OAUTH_TOKEN`.
 
+1. If you're setting up a [Slack Bridge][6], switch the value of the
+   `?api_key=` query in the **integration URL** you generated with the
+   API key of the **Generic bot** used for the Slack Bridge. Make sure
+   to continue the setup with this new URL. Otherwise, proceed to the
+   next step.
```

Comment:
I think it'd be better to have this at the start, before we ask you to make an incoming webhook bot. Seems annoying to get this far and then need to go make another bot. Something like:

> If you'd like to run a bidirectional bridge, you'll want to create a generic bot ...

Reply:
ah Ok, my thinking was the user has started the setup by reading the Slack bridge instruction in the python-zulip-api repo first which covers making a generic bot for the Slack bridge before setting up the Slack webhook integration.

Reply:
I move the instruction to just below step 4